src/Apps/Artist/Routes/Consign/Components/ArtistConsignHeader.tsx

```diff
@@ -46,3 +68,137 @@ export const ArtistConsignHeader: React.FC<ArtistConsignHeaderProps> = props =>
     </SectionContainer>
   )
 }
+
+/**
+ * Displays a random set of images for a given consignment artist.
+ *
+ * Some notes about implementation:
+ *
+ * 1) Due to responsiveness and SSR we need to dynamically scale the width and the
+ *    height of the image based on overall width of the container.
+ * 2) For this we use our `<ResponsiveImage>` component.
+ * 3) However, this component achieves responsivity by forcing padding within a
+ *    container and then applying a "fit" to a background image.
+ * 4) Therefore, conventional borders cannot be applied, and a CSS filter is
+ *    required. This is based upon pixel data, and border faked.
+ * 5) The pixel data comes from a second image, sitting below the first, with
+ *    its brightness turned to -100.
+ * 5) It's not perfect, and the layout needs to also account for variable
+ *    aspect ratios.
+ * 6) The positioning CSS is also tricky, so please forgive...
+ */
+const HeaderImages = (props: {
+  artworksByInternalID: Consign_artworksByInternalID
+}) => {
+  const getRandomImages = () => {
```

Comment:
This function seems worth getting some specs around if for no other reason than to help future us remember what it's doing in case we have to change it! 😝